packages/lesswrong/components/ea-forum/giving-portal/AdminElectionCandidates.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { Components, registerComponent } from "../../../lib/vulcan-lib";
+import { userIsAdmin } from "../../../lib/vulcan-users";
+import { useCurrentUser } from "../../common/withUser";
+import { useElectionCandidates } from "./hooks";
+import { Link } from "../../../lib/reactRouterWrapper";
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    maxWidth: 1200,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    gap: "20px",
+  },
+  link: {
+    color: theme.palette.primary.main,
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 16,
+    lineHeight: '22px',
+  },
+});
+
+const AdminElectionCandidates = ({classes}: {
+  classes: ClassesType,
+}) => {
+  const currentUser = useCurrentUser();
+  const {results, loading} = useElectionCandidates("name", {
+    fetchPolicy: "network-only",
```

Comment:
This is required to make sure we refetch the correct list after renaming or deleting a candidate.